Help message shouldn't depend on valid AWS profile

Just a small usability feature to make introspector easier to learn.

I don't have a default AWS profile in my environment.

When I set an AWS profile for the command the help message does appear.

```
$ AWS_PROFILE=introspector ./introspector account aws import --help
Usage: introspector.py account aws import [OPTIONS]

  Imports the assets from an AWS account into introspector

Options:
  -f, --force         Skip prompts for adding accounts
  -s, --service TEXT  Only import the specified service(s)
  -g, --gov-cloud     Set this flag to import a govcloud account
  --help              Show this message and exit.
```

I would expect the help message to appear without an AWS profile set, but instead it prints an error.

```text
$ ./introspector account aws import --help
panic: no EC2 IMDS role found, operation error ec2imds: GetMetadata, request canceled, context deadline exceeded

goroutine 1 [running]:
main.main()
	/Users/greg/workspace/introspector/launcher/main.go:208 +0x3d9
```

--- a/launcher/main.go
+++ b/launcher/main.go
@@ -17,6 +17,17 @@ import (
 	"github.com/pkg/errors"
 )
 
+type authError struct {
+	Err error
+}
+func (e *authError) Error() string {
+	return "Failed to find AWS Credentials"
+}
+
+func (e *authError) Unwrap() error {
+	return e.Err
+}
+ 
 func requireIntrospectorComposition(ctx context.Context, cli *client.Client) types.Container {
 	containers, err := cli.ContainerList(ctx, types.ContainerListOptions{
 		Filters: filters.NewArgs(filters.Arg("label", "introspector-cli")),
@@ -36,11 +47,11 @@ func requireIntrospectorComposition(ctx context.Context, cli *client.Client) typ
 func loadAwsCredentials(ctx context.Context) (map[string]string, error) {
 	cfg, err := config.LoadDefaultConfig(ctx)
 	if err != nil {
-		return nil, err
+		return nil, &authError{err}
 	}
 	creds, err := cfg.Credentials.Retrieve(ctx)
 	if err != nil {
-		return nil, err
+		return nil, &authError{err}
 	}
 	env := make(map[string]string)
 	env["AWS_ACCESS_KEY_ID"] = creds.AccessKeyID
@@ -205,7 +216,13 @@ func main() {
 	for _, cmd := range cmds {
 		err = cmdPassthrough(ctx, cli, introspector, cmd)
 		if err != nil {
-			panic(err)
+			var authErr *authError
+			if errors.As(err, &authErr) {
+				fmt.Println("Failed to find AWS Credentials. Please ensure that your enviroment is correctly configued as described here: https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-configure.html")
+				os.Exit(1)
+			} else {
+				panic(err)
+			}
 		}
 	}
 }

